src/main/resources/testchipip/csrc/plusarg_file_mem.h

```diff
@@ -0,0 +1,22 @@
+#ifndef __PLUSARG_FILE_MEM_H__
+#define __PLUSARG_FILE_MEM_H__
+
+class PlusargFileMem
+{
+  public:
+    PlusargFileMem(const char *filename, bool writeable, int capacity_words, int data_bytes);
+    ~PlusargFileMem(void);
+
+    void read(long long address, long long *data);
```

Comment:
Can we use fixed width types here if you have a specific width in mind.

Reply:
Sure, can do